coral/src/app/features/connectors/details/settings/ConnectorSettings.tsx

```diff
@@ -0,0 +1,204 @@
+        )}
+        {!highestEnv && (
+          <li>
+            The connector is on a higher environment. Please delete the
+            connector from that environment first.
+          </li>
+        )}
+        {hasOpenRequest && <li>The connector has a pending request.</li>}
+      </ul>
+    );
+  }
+
+  return (
+    <>
+      {showConfirmation && (
+        <ConnectorDeleteConfirmationModal
+          isLoading={isLoading}
+          onSubmit={(remark) => mutate(remark)}
+          onClose={() => setShowConfirmation(false)}
+        />
+      )}
+
+      <PageHeader title={"Settings"} />
+      {errorMessage && (
+        <Box role="alert" marginBottom={"l2"}>
+          <Alert type="error">{errorMessage}</Alert>
+        </Box>
+      )}
+
+      {!connectorOwner && (
+        <div>
+          Settings can only be edited by team members of the team the connector
+          does belong to.
+        </div>
+      )}
+
+      {connectorOwner && (
+        <>
+          <Typography.Subheading>Danger zone</Typography.Subheading>
+          {connectorIsRefetching && (
+            <div className={"visually-hidden"}>Loading information</div>
+          )}
+          {connectorIsRefetching && (
+            <BorderBox
+              date-testid={"connector-settings-danger-zone-content"}
+              display={"flex"}
+              flexDirection={"column"}
+              borderColor={"error-60"}
+              padding={"l2"}
+              marginTop={"l2"}
+              rowGap={"l2"}
+            >
+              <Box
+                display={"flex"}
+                alignItems={"center"}
+                justifyContent={"space-between"}
+              >
+                <Box width={"full"}>
+                  <Skeleton />
+                  <Skeleton />
+                </Box>
+                <div>
+                  {/* eslint-disable-next-line @typescript-eslint/no-empty-function */}
+                  <Button.Primary onClick={() => {}} disabled={true}>
+                    Request connector deletion
+                  </Button.Primary>
+                </div>
+              </Box>
+            </BorderBox>
+          )}
+
+          {!connectorIsRefetching && (
+            <BorderBox
+              date-testid={"connector-settings-danger-zone-content"}
+              display={"flex"}
+              flexDirection={"column"}
+              borderColor={"error-60"}
+              padding={"l2"}
+              marginTop={"l2"}
+              rowGap={"l2"}
+              aria-hidden={connectorIsRefetching}
+            >
+              {!showDeleteConnector && (
+                <Alert type={"warning"}>
+                  {connectorIsRefetching ? (
+                    <Typography.DefaultStrong htmlTag={"div"}>
+                      <Skeleton />
+                    </Typography.DefaultStrong>
+                  ) : (
+                    <>
+                      You can not create a delete request for this connector:{" "}
```

Comment:
Probably c&p from my error :D but the `{" "}` is not needed since we're doing a line break.

Reply:
Probably my auto formatter ^^